bots/task/github.py

```diff
@@ -45,6 +45,10 @@
 NOT_TESTED = "Not yet tested"
```

Comment:
To further explain what's going on, please add a comment here that this is a webhook-triggered test, as opposed to a user one.